sdk/dotnet/src/Microsoft.Datasync.Client.SQLiteStore/OfflineSQLiteStore.cs

```diff
@@ -213,7 +213,9 @@ public override async Task UpsertAsync(string tableName, IEnumerable<JObject> it
                 {
                     throw new InvalidOperationException($"Column '{prop.Name}' is not defined on table '{tableName}'");
                 }
-                columns.Add(column);
+                
+                if(column != null)
```

Comment:
Please add curly-braces around the if, and a space between the if and open-paren.  (Style consistency)

Reply:
Again, I'm quite new to this Git stuff. I made the changes in my fork but don't know if that's the full procedure.